docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.1"
+services:
+  ucurtma-container:
+    build:
+      context: ./
+      dockerfile: ./docker/alpine/Dockerfile
```

Comment:
folder structure seems quite good, but this `Dockerfile` seems to be customised for testing and doesn't really look like the bare `alpine` image. Can we rename the folder to be more descriptive of what that image is about?